Add a mention of CPU-only arguments inputs in docs (#3039)
NVIDIA/DALI: docs/supported_ops.rst

.. currentmodule:: nvidia.dali

> Operations functions are used to define the data processing graph within a DALI :ref:`Pipeline <pipeline>`.

Reviewer: This sentence doesn't sound like English. 
```suggestion
The data processing graph within a DALI :ref:`Pipeline <pipeline>` is defined by calling operation functions.
```

> They accept as inputs and return as outputs :class:`~nvidia.dali.pipeline.DataNode` instances, which represent batches of Tensors.

Reviewer: ```suggestion
They accept and return instances of :class:`~nvidia.dali.pipeline.DataNode`, which are _symbolic_
representations of batches of Tensors.
```

Author: done

> It is worth noting that those operation functions can not be used to process data directly.

Reviewer: ```suggestion
The operation functions **cannot** be used to process data directly.
```

Author: done

> The constraints for defining the processing pipeline can be find in :ref:`this section <processing_graph_structure>`

Reviewer: ```suggestion
The constraints for defining the processing pipeline can be found in :ref:`this section <processing_graph_structure>`
```

Author: done
